content/en/docs/Reference/immutable_rootfs.md

```diff
@@ -110,6 +110,8 @@ In the environment file few options are available:
   device. Default value is:
 
   `RW_PATHS="/etc /root /home /opt /srv /usr/local /var"`
+  **Note**: as those paths are overlayed with an ephemeral mount (`tmpfs`), 
```

Comment:
dont we have a note tag?

Reply:
tried it there but screws things quite a bit when I did the preview locally :/